### DISCUSSION TITLE
DataGrid hide Current Rows in pagination

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#2105] [Q&A]
Hey,

can I hide this information in the pagination section of a datagrid?

![image](https://user-images.githubusercontent.com/10349466/112482757-acd27e00-8d78-11eb-9be0-0f32a7baa7f4.png)

Kind regards

**Comment by stsrki** [comment]
Yes, just define `TotalItemsTemplate` and leave it empty

```cshtml
<DataGrid @ref="dataGrid">
  <TotalItemsTemplate>
    @* leave it empty *@
  </TotalItemsTemplate>
</DataGrid>
```

**Reply by bastigerner** [reply]
Thanks worked like a charm!

**Reply by bastigerner** [reply]
https://Ko-fi.com/home/coffeeshop?txid=cda9ca2c-e19b-4284-9a6d-23e4dcb7715f&mode=public&img=ogiboughtsomeone

😄

**Reply by stsrki** [reply]
Thank you 😎
